src/javascript/ContentEditor/ContentEditor/EditPanel/EditPanel.scss

```diff
@@ -42,6 +42,11 @@
     height: calc(100vh - 96px);
 }
 
+:root {
+    --ce-disabled-input-border: 1px solid var(--color-gray_dark);
+    --ce-disabled-input-background: var(--color-gray_light);
+}
```

Comment:
- You can use moonstone variables to manage border: https://github.com/Jahia/moonstone/blob/develop/src/globals/_variables.scss#L2
- The class `.moonstone-selector_disabled` will be available to the disabled style of moonstone selectors https://github.com/Jahia/moonstone/blob/develop/src/utils/_mixins.scss#L14-L22